Pax.sln

```diff
@@ -0,0 +1,59 @@
+﻿

```

Comment:
I agree that it's useful to preserve history. I wonder what "best 
practices" there are for doing this. The best idea I could come up with 
is to branch this PR into a different PR and carry out the cherrypicking 
there, and reference the second PR from the first PR, and close the 
first PR without merging, and continue developing the second PR.

On 2016-07-14 11:21, TMVector wrote:
> In Pax.sln [1]:
> 
>> @@ -0,0 +1,59 @@
>> +﻿
> 
> What's your opinion on changing commits in this PR? I think the
> comments on diffs would become orphaned?
> 
> --
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub [2], or mute the
> thread [3].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/niksu/pax/pull/9#discussion_r70782326
> [2]
> https://github.com/niksu/pax/pull/9/files/7f29135a9261e8b25f7b9c3ec6cc9f4b3b0add3a..f7ac43cbd665a50aff815bdb5eb1eb81e6b9d99b#r70782326
> [3]
> https://github.com/notifications/unsubscribe/ABDTTslJkLs2j3Kzo40WL2X0FeFPJegOks5qVg2zgaJpZM4JLXUX

-- 
http://www.cl.cam.ac.uk/~ns441/